### ISSUE TITLE
not compatible with Morfeusz 2

### ISSUE THREAD

**Issue by jwilk** [jwilk/python-morfeusz#1]
python-morfeusz is not compatible with Morfeusz 2.
Upstream changed SONAME (and probably also tagset):

``` pytb
Traceback (most recent call last):
  File "tests.py", line 11, in <module>
    import morfeusz
  File "/home/jwilk/python-morfeusz/morfeusz.py", line 125, in <module>
    libmorfeusz = ctypes.CDLL('libmorfeusz.so.0')
  File "/usr/lib/python2.7/ctypes/__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libmorfeusz.so.0: cannot open shared object file: No such file or directory
```

Given that upstream provides now their own Python bindings (with different API), I think we should just document the incompatiblity and deprecate this project.

If there are still users who want this project to be continued, please let me know.

**Comment by jwilk** [comment]
To clarify, by "upstream" I mean (maintainers of) Morfeusz proper.

**Comment by jwilk** [comment]
In 111f27eb8f36da8c8a696f2671c2bbb1e37c50d8, I've set up development status as *inactive* in setup.py.
In b11baf04fdeda8e9832a0cf34f357247a9279e9b, I added README that documents what's the last supported Python version, and points people to Morfeusz 2 own Python interface.
I intend to make a final release of python-morfeusz in a few days.

**Comment by jwilk** [comment]
I've released python-morfeusz 0.3400.
